docker/zip.bzl

```diff
@@ -0,0 +1,26 @@
+# Copyright 2017 The Bazel Authors. All rights reserved.
```

Comment:
We could put that in bazel directly but that would be the wrong direction. We could also create a new repo that would have pkg_tar, gzip and some more.